src/liblibc/lib.rs

```diff
@@ -1931,10 +1933,30 @@ pub mod types {
                 pub type PLONG = *mut c_long;
 
                 #[cfg(target_arch = "x86")]
+                pub type LONGLONG = c_double;
+                #[cfg(target_arch = "x86_64")]
+                pub type LONGLONG = i64;
+
+                pub type ULONG = c_ulong;
+
+                #[cfg(target_arch = "x86")]
+                pub type ULONGLONG = c_double;
```

Comment:
Ditto. I was expecting an unsigned double, but the specification declares just double